src/main/java/org/kpmp/geneExpressionSummary/Participant.java

```diff
@@ -0,0 +1,13 @@
+package org.kpmp.geneExpressionSummary;
```

Comment:
This interface is not really used anywhere, so it can be omitted. We have introduced interfaces in a few places so that we can treat the results from sc and sn searches as the same type of object, but that isn't needed here.